docs/guide/configuration/rotation_for_logfiles.md

```diff
@@ -28,7 +29,7 @@ Create a new configuration file for `logrotate` in `/etc/logrotate.d/silver-esho
 }
 ```
 
-If the logfiles are getting big very quickly, it's possible to run the `logrotate` once per hour by putting it in `cron.hourly` (it will run at 17 minutes every hour)
+If the logfiles grow very quickly, you can run `logrotate` once per hour by putting it in `cron.hourly` (it will run at 17 minutes every hour)
```

Comment:
```suggestion
If the logfiles grow very quickly, you can run `logrotate` once per hour by putting it in `cron.hourly` (it will run at 17 minutes past every hour):
```